tf/deployment/modules/scoped/monitoring/grafana/grafana.tf

```diff
@@ -2,23 +2,55 @@ resource "grafana_contact_point" "leadership_alerts" {
   name = "leadership-alerts"
 
   discord {
-    url = data.terraform_remote_state.discord_state.outputs.leadership_alerts_grafana_webhook_url
+    url     = data.terraform_remote_state.discord_state.outputs.leadership_alerts_grafana_webhook_url
+    message = "{{ template \"discord.message\" . }}"
+    title   = "{{ template \"discord.title\" . }}"
   }
 }
 
 resource "grafana_contact_point" "team_alerts" {
   name = "team-alerts"
 
   discord {
-    url = data.terraform_remote_state.discord_state.outputs.team_alerts_grafana_webhook_url
+    url     = data.terraform_remote_state.discord_state.outputs.team_alerts_grafana_webhook_url
+    message = "{{ template \"discord.message\" . }}"
+    title   = "{{ template \"discord.title\" . }}"
+  }
+}
+
+resource "grafana_notification_policy" "default" {
+  contact_point   = grafana_contact_point.team_alerts.name
+  group_by        = ["alertname"]
+  group_interval  = "1m"
+  group_wait      = "0s"
+  repeat_interval = "1h"
+
+  policy {
+    matcher {
+      label = "channel"
+      match = "="
+      value = "team"
+    }
+    contact_point = grafana_contact_point.team_alerts.name
+  }
+
+  policy {
+    matcher {
+      label = "channel"
+      match = "="
+      value = "leadership"
+    }
+    contact_point = grafana_contact_point.leadership_alerts.name
   }
 }
 
 module "example_dashboards" {
-  source = "./shared/modules/grafana"
+  # source = "/home/zack/Source/immich/devtools/tf/shared/modules/grafana"
```

Comment:
Stale comment

Reply:
Planning to leave this in so it's easy to switch to local dev when working on the module, and as an example for others of how to do it.